Sources/AloeStackView/AloeStackView.swift

```diff
@@ -187,6 +187,7 @@ open class AloeStackView: UIScrollView {
   /// If `animated` is `true`, the change is animated.
   open func setRowHidden(_ row: UIView, isHidden: Bool, animated: Bool = false) {
     guard let cell = row.superview as? StackViewCell else { return }
+    guard cell.isHidden != isHidden else { return }
```

Comment:
Style: put this extra condition in the first `guard` statement after a comma.

Reply:
Fixed 👍